docs/pbc/all/multi-factor-auth/202505.0/multi-factor-auth.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Multi-Factor Authentication
+description: Learn about Multi Factor Authentication (MFA) that you can use within your Spryker project.
+template: concept-topic-template
+last_updated: Mar 05, 2026
+---
+
+{% info_block warningBox "⚠️ WARNING: BETA FEATURE" %}
```

Comment:
Is this beta or early access as we had for dynamic multistore?

Reply:
beta